# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../mono/glibc-2.18.patch
# Copyright (C) 2020 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- mono-4.8.0/mono/io-layer/processes.c.vanilla	2020-02-20 21:04:13.943119193 +0000
+++ mono-4.8.0/mono/io-layer/processes.c	2020-02-20 21:04:44.179119036 +0000
@@ -45,6 +45,8 @@
 #include <utime.h>
 #endif
 
+#include <sys/sysmacros.h>
+
 /* sys/resource.h (for rusage) is required when using osx 10.3 (but not 10.4) */
 #ifdef __APPLE__
 #include <TargetConditionals.h>
